### ISSUE TITLE
Black frames in 120 fps RAW video mode 

### ISSUE THREAD

**Issue by brownshark1993** [mirsadm/motioncam#232]
Seems like my phone can record full resolution 120 fps video with FreeDCam, but when I use the 120 fps mode in Motion Cam I get some black frames. 60 fps and below works fine

**Comment by mirsadm** [comment]
Keep in mind FreeDCam is not capturing RAW frames (as far as I know). There are usually more limitations when capturing RAW.

**Comment by brownshark1993** [comment]
Yes, that may be the case here, but I found it strange that I get black frames instead of just dropped ones
